.fuel-core/configs/stateConfig.json

```diff
@@ -469,6 +469,7 @@
     }
   ],
   "contracts": [],
```

Comment:
I missed this field, would be interesting to see how it could be integrated with `launchTestNode`. Here's an [example](https://github.com/FuelLabs/fuel-core/blob/c2f5a3a64c0e792ef5390346ce0264696c95ea21/crates/chain-config/src/config/snapshots/fuel_core_chain_config__config__state__tests__snapshot_contract_with_state.snap#L8) of a state config with this array filled.

Reply:
@nedsalk, I also reflected on this configuration yesterday. 

I believe we can use it to start the node with the contract already deployed, rather than deploying it afterward.

Reply:
I think state and balance are good candidates, but I'd argue that we should not move everything and continue deploying as we do to dogfood the [deployment functionality](https://github.com/FuelLabs/fuels-ts/issues/2878#issuecomment-2265617725).